release_dashboard/lib/widgets/clean_release_button.dart

```diff
@@ -54,7 +56,12 @@ class _CleanReleaseButtonState extends State<CleanReleaseButton> {
             leftButtonTitle: 'No',
             rightButtonTitle: 'Yes',
             rightButtonCallback: () async {
-              _updateErrorMsg('Feature has not been implemented yet. Please use conductor clean of the CLI tool!');
+              try {
+                await conductor.cleanRelease(context);
+                // ignore: avoid_catches_without_on_clauses
+              } catch (error, stackTrace) {
```

Comment:
Is there a conductor exception we can catch here? We want to catch specific exceptions before doing a catch all as the catch all will hide release issues

Reply:
`errorToString` will display what kind of exception it is.

Reply:
Yes, but you're adding the lint to ignore the type.

Try something like

```
try {
  // work
} catch (Conductor Exception error, StackTrace stackTrace) {
  // this is expected, maybe we can try again?
  errorToString();
} catch (dynamic error, stacktrace) {
  // this was unexpected
errorToString('FATAL!!!'); 
}
```

Reply:
There might be something I missed, but `errorToString` is able to accept any kind of error, and display its corresponding type.
```
String errorToString(Object error, StackTrace stackTrace) {
  final StringBuffer buffer = StringBuffer();
  buffer.writeln('${error.runtimeType}:\n$error');
  buffer.writeln('');
  buffer.writeln('Stack Trace:\n$stackTrace');
  return buffer.toString();
}
```

If it were a `Conductor Exception`, it would display `ConductorException: Something is wrong`. If it were a `Exception` type, it would display `Exception: something is wrong`. Let me know if that works.